source/_components/ebusd.markdown

```diff
@@ -0,0 +1,123 @@
+---
+layout: page
+title: "ebusd"
+description: "The ebusd component allows the integration between eBUS heating system and Home Assistant."
+date: 2018-12-27 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ebusd.png
+ha_category: Ebusd
+ha_iot_class: "Local Polling"
+ha_release: "0.85"
+---
+
+Integration between [ebusd](https://github.com/john30/ebusd/) daemon for communication with eBUS heating systems, and homeassistant using sensor component.
+
+## {% linkable_title Configuration %}
+
+Enable the sensor by adding the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+ebusd:
+  host: 127.0.0.1
+  circuit: '700'
+  monitored_conditions:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.